Labs/AI and Machine Learning/Custom Vision Service/Custom Vision Service.md

```diff
@@ -34,7 +34,10 @@ The following are required to complete this hands-on lab:
 <a name="Resources"></a>
 ### Resources ###
 
-[Click here](https://topcs.blob.core.windows.net/public/cvs-resources.zip) to download a zip file containing the resources used in this lab. Copy the contents of the zip file into a folder on your hard disk.
+
+[Click here](https://a4r.blob.core.windows.net/public/cvs-resources.zip) to download a zip file containing the resources used in this lab. Copy the contents of the zip file into a folder on your hard disk.
```

Comment:
Accessibility issue. Should probably read `[Download zip file containing resources](url)` or similar

Reply:
Good catch!